content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -86,7 +86,7 @@ enforced/disallowed:
 		<tr>
 			<td>Capabilities</td>
 			<td>
-				Adding additional capabilities beyond the <a href="https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities">default set</a> must be disallowed.<br>
+				Adding additional capabilities beyond the <a href="https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities">default set (excluding NET_RAW)</a> must be disallowed.<br>
```

Comment:
This is hard to parse (at least for me). Would "Adding NET_RAW or capabilities beyond the ... default set ... must be disallowed" read easier?

Reply:
> If feasible, stick NET_RAW inside a <tt> HTML element.

I'll try it, but need to check the preview. In the past code format nested in tables didn't render well.